src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs

```diff
@@ -45,6 +45,14 @@ public void ThenTheUserIsAbleToManageTheCatalogueSolutionsSection()
             ThenTheCatalogueSolutionDashboardIsPresented();
         }
 
+        [Then(@"the User is able to manage the Additional Services section")]
+        public void ThenTheUserIsAbleToManageTheAdditionalServicesSection()
+        {
+            Test.Pages.OrderForm.ClickEditAdditionalServices();
+            
+        }
+
```

Comment:
can you move this step into AdditionalServices

Reply:
done